cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectDockerfile.java

```diff
@@ -155,8 +155,13 @@ private void connectorPluginsPreStage(PrintWriter writer, List<Plugin> plugins)
                     String artifactHash = Util.hashStub(mvn.getGroup() + "/" + mvn.getArtifact() + "/" + mvn.getVersion());
                     String artifactDir = plugin + "/" + artifactHash;
 
+                    // For handling custom repositories, we need to write custom Maven settings file
+                    String settingsFile = "/tmp/" + artifactHash + ".xml";
+                    String settingsXml = "<settings xmlns=\"http://maven.apache.org/SETTINGS/1.0.0\"><profiles><profile><id>download</id><repositories><repository><id>custom-repo</id><url>" + repo + "</url></repository></repositories></profile></profiles><activeProfiles><activeProfile>download</activeProfile></activeProfiles></settings>";
```

Comment:
We should XML entity encode `repo` here, just in case the user managed to include something like `</url></repository></repositories></profile></profiles><activeProfiles><activeProfile>download</activeProfile></activeProfiles></settings>`

Reply:
Yes, that's fine with me.

Reply:
@tombentley Ok, I added the escaping if you want to have another look.